### PR TITLE
update v1.1 strings for "zh-TW" locale (fixes issue #858)

### PR DESCRIPTION
**Pull request by cvan**
for issue #858

thanks, @Delphine!

### CODE REVIEW

File: app/src/main/res/values-zh-rTW/androidcomponents_strings.xml [hunk 1]
```diff
@@ -35,59 +35,59 @@
...
-	<ul> 
-		<li>請檢查伺服器是否流量過高或是暫時離線？請稍後再試。</li>
-		<li>您可以瀏覽其他網站嗎？請檢查電腦的網路連線。</li>
-		<li>您的電腦或網路是否受防火牆或 Proxy 保護？錯誤設定可干擾網路瀏覽。</li>
-		<li>問題是否依然存在？請諮詢您的網路管理員或互聯網供應商尋求協助。</li>
+	<p>指定的網站一直沒有回應，瀏覽器已停止等待。</p>
+	<ul>
+		<li>該網站可能暫時流量過高？請稍候再試。</li>
+		<li>無法瀏覽其它網站？請檢查網路連線。</li>
+		<li>電腦有開防火牆或 Proxy？請確定這些設定是否正確。</li>
```
Comment (cvan) [inline app/src/main/res/values-zh-rTW/androidcomponents_strings.xml]: I noticed `Proxy` was moved and changed to `Proxy？`. this look ok, @Delphine?

File: app/src/main/res/values/localpages.xml [hunk 2]
```diff
@@ -6,21 +6,21 @@
     <!-- IMPORTANT NOTE: If this file is updated anyhow, the STRs wiki must be updated accordingly:
     https://github.com/MozillaReality/FirefoxReality/wiki/L10n -->
 
-    <!-- This string is used as the title of the Private Browsing page that is opened when the
+	  <!-- This string is used as the title of the Private Browsing page that is opened when the
     Private Browsing Mode button from the tray is clicked. -->
-    <string name="private_browsing_title">Private Browsing</string>
+    <string name="private_browsing_title">隱私瀏覽模式</string>
```
Comment (cvan) [inline app/src/main/res/values/localpages.xml]: oops, this file should not be changed (!)

File: app/src/main/res/values-zh-rTW/androidcomponents_strings.xml (hunk 1 above)
Comment (Delphine) [reply app/src/main/res/values-zh-rTW/androidcomponents_strings.xml]: Yes that looks ok, they use English words for certain terms

File: app/src/main/res/values/localpages.xml (hunk 2 above)
Comment (Delphine) [reply app/src/main/res/values/localpages.xml]: I'm not sure I understand?
If you mean that "Private Browsing" shouldn't be translated: we translate it for all other products (but not sure you're talking about that?)

**Comment by Delphine** [comment]
Nice catch for the typo :) 
I think this is good on my side now! I'll let you merge when you want.

**Pull request merged**